workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Program.cs

```diff
@@ -47,6 +47,11 @@ private static int Main(string[] args)
                 Ipc.WriteError(Ipc.ErrorCode.UnknownGrpcError, e.ToString());
                 return ErrorExitCode;
             }
+            catch (SpatialOS.Platform.Common.NoRefreshTokenFoundException)
+            {
+                Ipc.WriteError(Ipc.ErrorCode.Unauthenticated, "");
```

Comment:
Any extra context to add here?

Reply:
Not a whole lot - just means you haven't `spatial auth login`, we catch this in the Editor and auto retry and display some context there.

Seemingly some calls throw a Grpc Exception and some throw this one 😕

Reply:
For reference, this is what it looks like in the Editor if you reauth: 

![image](https://user-images.githubusercontent.com/13353733/57131394-2d0e2580-6d94-11e9-9471-24665894b08c.png)

Reply:
looks good to me. What happens if the reauth fails?

Reply:
Then they will get a message asking them to run `spatial auth login` manually 😄 

```csharp
            foreach (var wrappedTask in manager.CompletedTasks.OfType<AuthTask>())
            {
                var result = wrappedTask.Task.Result;

                if (result.ExitCode == 0)
                {
                    Debug.Log("Successfully authenticated with SpatialOS. Retrying previous action.");
                }
                else
                {
                    Debug.LogError("Failed to authenticate with SpatialOS. Please run \"spatial auth login\" manually.");
                }
            }
```

Although it may get stuck in a loop..

Reply:
Fixed the potential loop.

Reply:
Also we no print the `Debug.LogError` with the empty message